packages/wasm/wasm.0.1/url

```diff
@@ -0,0 +1,2 @@
+archive: "https://github.com/WebAssembly/spec/archive/0xd.tar.gz"
```

Comment:
The repository calls this version a pre-release with identifier `0xd` but this package is marked as version 0.1. What happens if the upstream *also* releases a 0.1 version?